### PR TITLE
Improve wording and explanation on the console porting process

### LINKED ISSUE #5625: Console Exports: Due to the chosen wording, people mistakenly think out of the box support for consoles is impossible
**Your Godot version:** 3.4.2.stable

**Issue description:**
I've had this conversation now ... I don't know how many times. It's one of these things that come up like clockwork in the community channels:

A lot of people mistakenly seem to think Godot will never have or cannot have out of the box support for consoles like Unity or Unreal does due to Godot being open source.

This is of course completely wrong and merely a matter of incentive and ironing out relevant details.

It seems to me a lot of these people get the idea from this part of the docs page which I am often linked to in these discussions: 

> The reason other consoles are not officially supported are:
> 
>     To develop for consoles, one must be licensed as a company. As an open source project, Godot does not have such a legal figure.
>     Console SDKs are secret and covered by non-disclosure agreements. Even if we could get access to them, we could not publish the platform-specific code under an open source license.
>     Consoles require specialized hardware to develop for, so regular individuals can't create games for them anyway.

While they are not factually wrong, these point makes it apparently sound to many as if there is no legal possible way for Godot users to have the same convenience and cost saving of premade export templates like Unity or Unreal users have.
The text tries to explain this, but phrases like 

>we could not publish the platform-specific code under an open source license.

make is sound as if releasing platform-specific (=proprietary) code under open source license would be a necessity in order to provide Godot users the same console support as Unity or Unreal users. [This is obviously not the case.](https://www.reddit.com/r/godot/comments/svp9ll/comment/hxp3ueu/?utm_source=share&utm_medium=web2x&context=3). 

At no point Godot would ever have to open source proprietary code in order to get the same console support as Unity or Unreal. Of course there are a lot of other questions that need to be answered and probably also legal details to be discussed if console manufactures in fact have the interest to partner with Godot in some shape or form. Such as who pays for the development and maintenance of the export templates and where are they hosted. But it's far from impossible like the doc page seem to suggest for many readers.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/tutorials/platform/consoles.html#official-support

### PR DESCRIPTION
**Pull request by golddotasksquestions**
Remove the official support section, add explanation on export templates
fixes https://github.com/godotengine/godot-docs/issues/5625

### CODE REVIEW

**Comment by pycbouh** [comment]
If we are removing the official support section, then further restructuring needs to happen. Both current and this version of the docs starts with an explanation of the process, when it should start with an introduction to the state of console export in Godot. Then it can describe the process to further elaborate on the situation. In other words, it needs to be made clear what the current state is, and with this PR references to it are drowned by technical explanation about the process.

I also think that the mention to message Juan is out of place. That bit is speculative and doesn't inform the people who are going to read it about anything useful. At the very least, this should only be added if PLC wants to advertise this search for certified developers.

**Comment by golddotasksquestions** [comment]
The situation is that most people have no idea, very little knowledge or misinformed idea about publishing on consoles.
The regular discussions on r/gamedev, r/godot and discord are really terrible and I was actually very thankful when this was added right at the start.
I also think the current state is sufficiently described after the general process is explained.

**Comment by pycbouh** [comment]
The article is not about publishing for consoles in general. It's about publishing for consoles in Godot. Thus this point should be made clear with the opening paragraph. The article should start with addressing the core subject in short and concise way, not with going on a tangent about hardships of making a console port.

> I also think the current state is sufficiently described after the general process is explained.

With the current formatting the most important information (what's the state of console support in Godot) is hidden by a wall of tangential text. It is a useful tangential text, but it's not the important part of the article. It is only there to give context. Godot documentation should not be your source of general game development information, same as it shouldn't be your general source for learning programming. It should be about Godot. Everything else should either be offloaded to third-party resources, or be made less prominent. So the structure suggested by this PR needs to change.

**Comment by golddotasksquestions** [comment]
>Godot documentation should not be your source of general game development information

This sounds to me like a really narrow minded mindset. 

There can be a middle ground imho, which is explaining a bit of the basics is when it is necessary and useful or consistently frequent questions or discussions on the community channels show it would help. Just like a bit of vector math basics is [explained here](https://docs.godotengine.org/en/stable/tutorials/math/vector_math.html) is indispensable helpful for any beginner game devs, so is a short paragraph about the basics of porting to consoles incredibly valuable to beginners for those who have no clue.

There are lot's of examples like that, and quite frankly I want more of it, not less. Godot attracts a lot of beginners. It advertises itself as "easy to program" and generally is perceived as the underdog light beginner friendly general purpose game engine compared to Unity, Unreal and others. Godot should continue to be accessible and welcoming to these beginners, and this starts with how we word our documentation.

**Comment by pycbouh** [comment]
> This sounds to me like a really narrow minded mindset.

This is a practical decision. We are not and cannot be expected to be experts in everything gamedev and creating and maintaining general purpose subjects is an additional task on top of maintaining Godot documentation. We are but a few though.

I'm not saying we shouldn't mention those topics, but it's much more reasonable to link to great third party articles on the matter and spend our limited capacity on things we are directly responsible for.

Regardless of your take on this decision, this is not related to your PR which has several review points which need to be addressed. I'm not asking you to get rid of the general purpose information. I'm asking to fix the structure of the document, so it can be followed more easily and so it gives an obvious answer to the question in the title of the article.

File: tutorials/platform/consoles.rst [hunk 1]
```diff
@@ -16,43 +16,44 @@ to a console platform is as follows:
...
-  developers (*devkit*). The cost of those devkits is confidential.
-- Port the engine to the console platform or pay a company to do it.
+- Get access to developer hardware and software (also known as devkit or SDK = 
+  Software development Kit).
+  The cost of those devkits is confidential.
+  The SDK hardware is usually a console specially made for developers. The 
+  software consists of tools and APIs which have to be integrated into your game 
+  and build process. 
+- Port the engine to the console platform or pay a company to do it. Porting is 
+  the porcess of integrating the consoles manufactors software and requirements
```
Comment (HopefulFrog) [inline tutorials/platform/consoles.rst:26]: Typo here with the word "process".

File: tutorials/platform/consoles.rst [hunk 2]
```diff
@@ -16,43 +16,44 @@ to a console platform is as follows:
...
-- Port the engine to the console platform or pay a company to do it.
+- Get access to developer hardware and software (also known as devkit or SDK = 
+  Software development Kit).
+  The cost of those devkits is confidential.
+  The SDK hardware is usually a console specially made for developers. The 
+  software consists of tools and APIs which have to be integrated into your game 
+  and build process. 
+- Port the engine to the console platform or pay a company to do it. Porting is 
+  the porcess of integrating the consoles manufactors software and requirements 
+  into the game and build process. There countless other smaller and bigger
```
Comment (HopefulFrog) [inline tutorials/platform/consoles.rst:27]: Missing "are" between "there" and "countless".

File: tutorials/platform/consoles.rst [hunk 3]
```diff
@@ -16,43 +16,44 @@ to a console platform is as follows:
...
+- Get access to developer hardware and software (also known as devkit or SDK = 
+  Software development Kit).
+  The cost of those devkits is confidential.
+  The SDK hardware is usually a console specially made for developers. The 
+  software consists of tools and APIs which have to be integrated into your game 
+  and build process. 
+- Port the engine to the console platform or pay a company to do it. Porting is 
+  the porcess of integrating the consoles manufactors software and requirements 
+  into the game and build process. There countless other smaller and bigger 
+  consideration and reuirements to be fullfilled (think Steam capsules), and
```
Comment (HopefulFrog) [inline tutorials/platform/consoles.rst:28]: "considerations and requirements"

File: tutorials/platform/consoles.rst [hunk 4]
```diff
@@ -16,43 +16,44 @@ to a console platform is as follows:
...
+  Software development Kit).
+  The cost of those devkits is confidential.
+  The SDK hardware is usually a console specially made for developers. The 
+  software consists of tools and APIs which have to be integrated into your game 
+  and build process. 
+- Port the engine to the console platform or pay a company to do it. Porting is 
+  the porcess of integrating the consoles manufactors software and requirements 
+  into the game and build process. There countless other smaller and bigger 
+  consideration and reuirements to be fullfilled (think Steam capsules), and 
+  adjustment to the controlls of your game to be made to support the specific
```
Comment (HopefulFrog) [inline tutorials/platform/consoles.rst:29]: "adjustments to the controls"

File: tutorials/platform/consoles.rst [hunk 5]
```diff
@@ -16,43 +16,44 @@ to a console platform is as follows:
...
-- Console SDKs are secret and covered by non-disclosure agreements.
-  Even if we could get access to them, we could not publish
-  the platform-specific code under an open source license.
-- Consoles require specialized hardware to develop for, so regular individuals
-  can't create games for them anyway.
-
-However, it is still possible to port your games to consoles thanks to services
-provided by third-party companies.
+Due to the complexity of the process, many studios and developers decide to 
+outsource console porting (see thrid-party providers below).
```
Comment (HopefulFrog) [inline tutorials/platform/consoles.rst:38]: "third-party"

**Review by HopefulFrog — COMMENTED** [review]
Found a few typos.

**Comment by mhilbrunner** [comment]
Superseded and merged in https://github.com/godotengine/godot-docs/pull/5764. Thanks for contributing!

**Pull request closed without merging**